testing/test_HeMIS.py

```diff
@@ -23,7 +23,7 @@
 BN = 0.1
 N_EPOCHS = 10
 INIT_LR = 0.01
-PATH_BIDS = 'testing_data'
+PATH_BIDS = 'testing_data/'
```

Comment:
now windows compatible-- shouldn't we use os.pathsep instead?

Reply:
You mean windows subsystem for linux? 
If not, I can track other similar issue.

Reply:
if in one day, ivadomed works with windows, this will break-- in general, it is good practice to use OS-agnostic separators in code

Reply:
i should also add, that you should not need to add a separator, if down the line paths are constructed in a pythonic way (e.g. using pathlib or os.path.join(), which should always be the case)

Reply:
Oh sorry I think I understood the first command wrong. The separator will be removed during cleaning (because it is not needed, I just changed the data path and I didn't pay cloe enough attention when I changed it back).